### PR TITLE
fix(instrumentation): CLI-335 add missing known commands

### PR DESCRIPTION
**Pull request by mgyorke**
Extended the list of known commands based on the [docs](https://docs.snyk.io/snyk-cli/cli-commands-and-options-summary).

### CODE REVIEW

File: pkg/instrumentation/categories.go [hunk 1]
```diff
@@ -79,7 +79,24 @@ func DetermineCategoryFromArgs(args []string, knownCommands []string, flagsAllow
...
+	"describe",
+	"container",
+	"debug",
+	"config",
+	"auth",
+	"ignore",
+	"log4shell",
+	"policy",
+	"fix",
+	"apps",
```
Comment (PeterSchafer) [inline pkg/instrumentation/categories.go:97]: Question: any reason the commands below are missing?
* iac capture
* iac rules init
* iac rules push
* iac update-exclude-policy

Comment (mgyorke) [reply pkg/instrumentation/categories.go:97]: The logic for building the category array parses the command line one argument at a time, it doesn't check "iac capture" as a single command. 

I'm assuming that this slice is used for that purpose only. If we need this, we also need to tweak the DetermineCategoryFromArgs method. 

Does this make sense?

Comment (PeterSchafer) [reply pkg/instrumentation/categories.go:97]: I understand that the list of known_commands contains just the individual command names not set in relation. That makes sense. 
My question is more about why the names "capture" or "init" are not there but the "rules" or "describe". Everything in this list is supposed to be known commands, so why not add all subcommands?

Comment (mgyorke) [reply pkg/instrumentation/categories.go:97]: Ah! ..you're right, nvm. I pushed the missing commands.

Comment (PeterSchafer) [reply pkg/instrumentation/categories.go:97]: Perfect, thanks!

**Pull request merged**